components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -651,10 +651,9 @@ interface KeyringServiceObserver {
 };
 
 enum OnRampProvider {
-  kWyre = 0,
-  kRamp = 1,
-  kSardine = 2,
-  kTransak = 3
+  kRamp = 0,
```

Comment:
I haven't found usage if this ids in persistent storages. Otherwise identifiers should be kept.